### ISSUE TITLE
props for raw variables?

### ISSUE THREAD

**Issue by stchang** [stchang/typed-rosette#18]
I think raw variables currently don't get props right?

Can I add it by doing something like:
```
  (define (var-assign/orig-binding x seps τs)
    (attachs
     (attachs (attach x 'orig-binding x)
              seps
              τs
              #:ev (current-type-eval))
     '(prop+ prop-)
     #`((Prop/ObjNotType #,x : CFalse)
        (Prop/ObjType #,x : CFalse))))
```

The scoping of `x` isn't quite right but is this on the right track?

**Comment by AlexKnauth** [comment]
Yes. Three things though. (1) You'll need to add `#:ev (current-type-eval)` to the second call as well. (2) Should it be `CFalse` here or does it need to be `False`? (3) What happens when a type rule specifies that a variable should have a `prop+` or `prop-` property? Do we want this to override that? Should we switch the order of the two `attachs` calls?

**Comment by stchang** [comment]
ok thanks, let me try to make it work.
